docs/topics/http/urls.txt

```diff
@@ -88,6 +88,9 @@ Notes:
   ``<int:name>`` to capture an integer parameter. If a converter isn't included,
   any string, excluding a ``/`` character, is matched.
 
+* ValueError raised by Converter(`Registering custom path converters`_) causes
+  HTTPError 404.
```

Comment:
I think we should move this note to the [registering-custom-path-converters](https://docs.djangoproject.com/en/2.2/topics/http/urls/#registering-custom-path-converters), e.g.
```
 * A ``to_python(self, value)`` method, which handles converting the matched
   string into the type that should be passed to the view function. It should
  raise ``ValueError`` if it can't convert the given value, which means no
  match and as a consequence a 404 response.
```

Reply:
Done.